baseline_models/models/aquatics_persistenceRW.R

```diff
@@ -39,26 +42,19 @@ RW_forecasts_EFI <- RW_forecasts %>%
          model_id = "persistenceRW") %>%
   select(model_id, datetime, reference_datetime, site_id, family, parameter, variable, prediction)
 
-#RW_forecasts_EFI |>
-#  filter(variable == "chla") |>
-#  ggplot(aes(x = time, y = prediction, group = ensemble)) +
-#  geom_line() +
-#  facet_wrap(~site_id)
-
-
-
 # 4. Write forecast file
 
 file_date <- RW_forecasts_EFI$reference_datetime[1]
 
+## TODO: does this need to be renamed?
 forecast_file <- paste("aquatics", file_date, "persistenceRW.csv.gz", sep = "-")
 
 RW_forecasts_EFI <- RW_forecasts_EFI |>
   filter(variable != "ch")
 
-
 write_csv(RW_forecasts_EFI, forecast_file)
 
+# TODO: need a new submission script
 neon4cast::submit(forecast_file = forecast_file,
                   metadata = NULL,
                   ask = FALSE)
```

Comment:
@rqthomas and @cboettig I'm curious why all the challenges have separate submit functions - e.g., [tern4cast](https://github.com/eco4cast/tern4cast/blob/e3e99aa553f583289aecf542070517ca2bfdd134/models/run_terrestrial_baselines.R#L175-L179), [vera4cast](https://github.com/LTREB-reservoirs/vera4castHelpers/blob/c15e291f7db0fa3cea306c828cc465b305ffe4f8/R/submit.R) and [neon4cast](https://github.com/eco4cast/neon4cast/blob/4b091c807e99a5e93ddc9a71c64ab233388abbaf/R/submit.R).   What do you think about creating a challenge-agnostic submit function rather than having this custom for each challenge?

Reply:
the neon4cast::submit() will work for usgsrc4cast project_id since they both are going to the same submission bucket.

Reply:
They are separate because they route to different s3 buckets.  A more general function would be ideal but not necessary for you to get started.

Reply:
but then there will be multiple `aquatics-[date]-persistenceRW.csv.gz` files that will overwrite each other in the `neon4cast-submissions`, right? or we'd need a separate theme for the usgsrc4cast challenge to not overwrite files ?

Reply:
Great point - I think you are correct that it would overwrite.  Maybe have the model_id slightly different for now.?  I think I might need to alter the submit and processing to have subdirectory for each project_id.

Reply:
yeah I think having a subdirectory for project_id should work. 

I looking at other `neon4cast` functions now and helper functions to download GEFS, for example, would have to be updated to if we want to use it for other forecast challenges. What do you think about making a new repo that is based on `neon4cast` but is challenge agnostic?

Reply:
I think this is a great idea.  It would take some discussion about how flexible we want the functions.  For example, are we assuming that all submissions go to the same S3 bucket?

Reply:
yeah probably same bucket for submission but different paths/subdir based on the challenge. Drivers, forecasts, archives, etc... are already following that pattern somewhat. e.g., https://github.com/eco4cast/usgsrc4cast-ci/blob/0f8c43e013d329fd0b7ac486f5a194ece8aa6543/challenge_configuration.yaml#L12-L23

Reply:
@rqthomas for a quick fix, I added the functions from `neon4cast` to the `usgsrc4cast` repo and modified them to be challenge agnostic https://github.com/eco4cast/usgsrc4cast-ci/tree/abebd84154306ceec56506c706c5bf5c7bb1cca2/R/eco4cast-helpers . Basically, they just require an additional input of the `project_id` to create a submission path that is a sub-directory of the submission bucket. 

I also slightly changed the name of the climatology and random walk models so they don't have the same object name as the `neon4cast` baseline models. https://github.com/eco4cast/usgsrc4cast-ci/blob/abebd84154306ceec56506c706c5bf5c7bb1cca2/baseline_models/models/aquatics_climatology.R#L117 

The submission processing for neon4cast https://github.com/eco4cast/neon4cast-ci/blob/main/submission_processing/process_submissions.R might need to be updated to filter out the non-neon forecasts in the submit bucket. I think a temporary fix would just be to filter out the `usgsrc4cast` submissions using a similar `str_detect(negate = T)` pattern that you have here https://github.com/eco4cast/neon4cast-ci/blob/70dd53e44d364591e8161d595029b7cb25666e0a/submission_processing/process_submissions.R#L44

Reply:
I added the `str_detect` and it is working.

Reply:
Cool. I think I'll merge this in then and continue pushing to the challenge-specific submission bucket.  I'll create a new issue for starting a separate forecast challenge-agnostic R package at some point